server.py

```diff
@@ -0,0 +1,49 @@
+# USAGE
+# python server.py --prototxt MobileNetSSD_deploy.prototxt --model MobileNetSSD_deploy.caffemodel --montageW 2 --montageH 2
+
```

Comment:
maybe rename this file to imagezmq_server or something, just if it's gonna be in the repo with the other code bc this isn't our server ya know

Reply:
good point- sorry forgot that it was in there still when i did the pr. thought i had cleaned up everything.